Source/JavaScriptCore/inspector/remote/RemoteInspector.cpp

```diff
@@ -101,10 +97,8 @@ void RemoteInspector::unregisterTarget(RemoteControllableTarget* target)
     pushListingsSoon();
 }
 
-void RemoteInspector::updateTarget(RemoteControllableTarget* target)
+void RemoteInspector::updateTarget(Ref<RemoteControllableTarget> target)
```

Comment:
Should not be a smart pointer. There is no ownership transfer here, the call site is in charge of protecting the object it is passing.